perl-package/AI-MXNet/t/test_kvstore.t

```diff
@@ -73,6 +75,93 @@ sub test_list_kv_pair
     }
 }
 
+sub test_row_sparse_pull
```

Comment:
Does it include the test case added a few days ago https://github.com/apache/incubator-mxnet/pull/9887/files ?

Reply:
yes, I added that test.